### PR TITLE
pipeline_ds_a1

### PR DESCRIPTION
**Pull request by philbosch**
pipeline for dataset a1 (medizinisch_technische_infrastruktur)
currently without ingestion into the postgres db and only a first idea of renaming variables. Open for suggestions
thanks a lot!

### CODE REVIEW

File: scripts/A1.R [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+# rename variables --------------------------------------------------------
+
+
+ds$data %>%
+  dplyr::rename(
+    "genutzte_infrastruktur" = infrastruktur,
+    "geraet_oder_untersuchung" = gerate_und_untersuchungen,
+     "raeumliche_ebene" = grossregion_kanton,
```
Comment (sabinem) [inline scripts/A1.R:86]: I would suggest that this column should be named "spatialunit_uid" : same name as in the postgres table.

File: scripts/test_ds.R
Comment (sabinem) [inline scripts/test_ds.R:1]: Can you please restore this file to its previous version: this should not be part of the PR.

**Review by sabinem — CHANGES_REQUESTED** [review]
Hello Philipp,

Your method of mapping the spatial unit seem pragmatic, but since it work, it's fine for me for now. 

I have just two change requests:

- `test_ds`: does not belong to the pipeline and should not be changed in this PR
- `the transformed datasets column should match to the postgres table, which has the column "spatialunit_uid".

File: scripts/test_ds.R
Comment (philbosch) [reply scripts/test_ds.R:1]: Good point! Thanks for the catch. Should I just delete the file, as it was just to test the pipeline and is not used in production any more?

File: scripts/A1.R (hunk 1 above)
Comment (philbosch) [reply scripts/A1.R:86]: Right now the data also includes a column "spatialunit_uid" and "raeumliche_ebene" is somewhat redundant. Would you agree if we delete "raeumliche_ebene"? 

This is how the data looks like:

grafik

Comment (sabinem) [reply scripts/A1.R:86]: Yes I agree that it makes sense to delete "raeumliche_ebene"

File: scripts/test_ds.R
Comment (sabinem) [reply scripts/test_ds.R:1]: Yes I think you can delete the file since we now already have other examples.

**Review by sabinem — APPROVED** [review]
